### PR TITLE
fix missing common properties in postgresql module

### LINKED ISSUE #2182: Missing common properties in postgresql module
Missing common properties in postgresql module after refactoring

### PR DESCRIPTION
**Pull request by antarus**
close #2182

### CODE REVIEW

**Comment by antarus** [comment]
@DanielFran i don't see error in Integration Tests..
I add cucumber test.

**Comment by DanielFran** [comment]
@antarus Yes I know, because there is no impacts calling or not the postgresservice on those tests...
I identify the issue when I was migrating the mariadb service, in fact calling only the service did not apply changes in the project, you can validate in temporary folder.

File: src/test/features/postgresql.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      | /api/servers/spring-boot/databases/postgresql |
+    Then I should have files in ""
+      | pom.xml |
+    And I should have history entry for "postgresql"
+    And I should have files in "documentation"
+      | postgresql.md |
+    And I should have files in "src/main/docker"
+      | postgresql.yml |
+    And I should have files in "src/main/java/tech/jhipster/chips/technical/infrastructure/secondary/postgresql"
+      | DatabaseConfiguration.java |
```
Comment (DanielFran) [inline src/test/features/postgresql.feature:16]: Missing specific dialect class and respective test class

File: src/test/features/postgresql.feature [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      | baseName    | jhipster            |
+    Then I should have files in ""
+      | pom.xml |
+    And I should have history entry for "postgresql"
+    And I should have files in "documentation"
+      | postgresql.md |
+    And I should have files in "src/main/docker"
+      | postgresql.yml |
+    And I should have files in "src/main/java/tech/jhipster/chips/technical/infrastructure/secondary/postgresql"
+      | DatabaseConfiguration.java |
```
Comment (DanielFran) [inline src/test/features/postgresql.feature:42]: Idem

**Review by DanielFran — CHANGES_REQUESTED** [review]

**Comment by antarus** [comment]
@DanielFran  

Sorry, but I don't find  any error.
I generated an application with
- init
- maven-java
- java-base
- springboot
- postgresql

After
`./mvnw clean verify`

no errors

i add 
- liquibase and `./mvnw clean verify` => no errors
- liquibase-user-and-authority-changelogs and `./mvnw clean verify` => no errors

I would like some help to point out the error.

I will update the cucumber tests

**Comment by antarus** [comment]
I think I understood what you were asking me for the integration tests

**Review by DanielFran — APPROVED** [review]

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java [hunk 3]
```diff
@@ -53,6 +53,36 @@ void shouldCreateModule() {
       .containing("<artifactId>postgresql</artifactId>")
       .containing("<groupId>com.zaxxer</groupId>")
       .containing("<artifactId>HikariCP</artifactId>")
-      .containing("<groupId>org.hibernate</groupId>");
+      .containing("<groupId>org.hibernate</groupId>")
```
Comment (DanielFran) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java:56]:
```suggestion
      .containing("<groupId>org.hibernate</groupId>")
      .containing("<artifactId>hibernate-core</artifactId>")
      .containing("<groupId>org.testcontainers</groupId>")
      .containing("<artifactId>mariadb</artifactId>")
```

**Review by DanielFran — CHANGES_REQUESTED** [review]

**Review by DanielFran — APPROVED** [review]

**Pull request merged**